resources/views/components/references.blade.php

```diff
@@ -104,38 +104,52 @@
+                    class="flex items-center gap-x-4 gap-y-4 md:border-t md:border-gray-900/10 px-6 py-4 sm:flex-nowrap">
                     <img class="h-10 w-10 flex-none rounded-full bg-gray-50"
                          src="{{ $references[$main]->getPhotoPath() }}"
                          alt="{{ __('alt.reference') . " " . $references[$main]->creator_name }}">
                     <div class="flex-auto">
                         <div class="font-semibold text-left">{{ $references[$main]->creator_name }}</div>
-                        <div class="text-gray-600 text-left">{{ $references[$main]->company }}</div>
+                        <div
+                            class="text-gray-600 text-left">
+                            @isset( $references[$main]->position ){{ $references[$main]->position }}@endisset
+                            @if( isset($references[$main]->position) && isset($references[$main]->company) ){{ __('at') }}@endif
+                            @isset( $references[$main]->company ){{ $references[$main]->company }}@endisset
+                        </div>
                     </div>
                 </figcaption>
             </figure>
         @endisset
         @isset($main2)
             <figure class="rounded-2xl bg-white shadow-lg ring-1 ring-gray-900/5">
-                <blockquote class="p-6 md:text-lg md:font-semibold leading-7 md:tracking-tight text-gray-900 sm:p-12 sm:text-xl md:leading-8">
+                <blockquote
+                    class="p-6 md:text-lg md:font-semibold leading-7 md:tracking-tight text-gray-900 sm:p-12 sm:text-xl md:leading-8">
                     <p>{!! "“".$references[$main2]->description."”" !!}</p>
                 </blockquote>
-                <figcaption class="flex flex-wrap items-center gap-x-4 gap-y-4 md:border-t md:border-gray-900/10 px-6 py-4 sm:flex-nowrap">
+                <figcaption
+                    class="flex flex-wrap items-center gap-x-4 gap-y-4 md:border-t md:border-gray-900/10 px-6 py-4 sm:flex-nowrap">
                     <img class="h-10 w-10 flex-none rounded-full bg-gray-50"
                          src="{{ $references[$main2]->getPhotoPath() }}"
                          alt="{{ __('alt.reference') . " " . $references[$main2]->creator_name }}">
                     <div class="flex-auto">
                         <div class="font-semibold text-left">{{ $references[$main2]->creator_name }}</div>
-                        <div class="text-gray-600 text-left">{{ $references[$main2]->company }}</div>
+                        <div class="text-gray-600 text-left">
+                            @isset( $references[$main2]->position ){{ $references[$main2]->position }}@endisset
+                            @if( isset($references[$main2]->position) && isset($references[$main2]->company) ){{ __('at') }}@endif
+                            @isset( $references[$main2]->company ){{ $references[$main2]->company }}@endisset
+                        </div>
```

Comment:
```suggestion
                            @isset($references[$main2]->position) {{ $references[$main2]->position }} @endisset
                            @if(isset($references[$main2]->position) && isset($references[$main2]->company)) {{ __('at') }}@endif
                            @isset($references[$main2]->company) {{ $references[$main2]->company }} @endisset
                        </div>
```

Reply:
These block are repeated. Can you make a component from that?

Reply:
Also: are we safe when there'd be no `$main2` index in `$references`?

Reply:
> Also: are we safe when there'd be no `$main2` index in `$references`?

Yes, we are handling that.